courses/free-courses-en.md

```diff
@@ -769,6 +770,16 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://coursesity.com/course-detail/free-advanced-jenkins-in-k8s-docker-in-docker?ref=share-link) - coursesity (Google)
+* [Getting Started with Jenkins](https://www.simplilearn.com/learn-jenkins-basics-free-course-skillup) - simplilearn (Google)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Automation Step by Step (YouTube)
+* [Jenkins Full Course | Jenkins Tutorial For Beginners | Jenkins Tutorial | Simplilearn](https://youtu.be/FX322RVNGj4) - Simplilearn (YouTube)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links